chef/data_bags/crowbar/template-monasca.json

```diff
@@ -24,7 +24,8 @@
         "max_buffer_size": 1000,
         "max_measurement_buffer_size": -1,
         "backlog_send_rate": 1000,
-        "amplifier": 0
+        "amplifier": 0,
+        "skip_enable": true
```

Comment:
I wonder if, assuming that service file is created in RPM package, this variable should be added here at all.
Why just not enforce it ?